source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,332 @@ Sorts Builders
    :class: singlecol
 
 .. _sorts-builders:
+
+Overview
+--------
+
+In this guide, we show you how to specify **sort criteria** for your
+queries using **builders**.
+
+Sort criteria are the rules MongoDB uses to sort your data. Some
+examples of sort criteria are:
+
+* Smallest number to largest number
+* Earliest time of day to latest time of day
+* Alphabetical order by first name 
+
+The builder pattern allows you to create complex immutable objects
+separately from their representation. :doc:`Builders </fundamentals/builders/>` 
+are classes provided by the MongoDB Java driver that allow you to use
+the builder pattern to construct
+:java-docs:`Bson <apidocs/bson/org/bson/conversions/Bson.html>` objects.
+The builders provided by the MongoDB Java driver can improve the
+readability of your code.
+
+You should read this guide if you would like to:
+
+* Use the builder pattern to specify sort criteria for your queries.
+* Perform ascending sorts and descending sorts.
+* Combine sort criteria.
+* Sort on the text score of a
+  :manual:`text search </core/text-search-operators/>`.
+
+If you want to learn the fundamentals of sorting in the MongoDB Java
+driver, consider reading our
+:doc:`guide on sorting </fundamentals/crud/read-operations/sort/>`.
+
+The Sorts Class
+---------------
+
+The :java-docs:`Sorts </apidocs/mongodb-driver-core/com/mongodb/client/model/Sorts.html>`
+class is a builder that provides static factory methods for all sort criteria
+operators supported by MongoDB. These methods return a
+:java-docs:`Bson </apidocs/bson/org/bson/conversions/Bson.html>`
+object that you can pass to the 
+:java-docs:`sort() </apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#sort(org.bson.conversions.Bson)>`
+method of a ``FindIterable`` instance. 
+
+.. _sorts-builders-sort-example:
+
+The following examples show you how to use the methods
+provided by the ``Sorts`` class to sort your queries. The examples use a
+sample collection, ``sort_example``, that contains the following documents:
+
+.. code-block:: json
+
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+
+
+Sorting Direction
+-----------------
+
+The ``Sorts`` class provides methods for specifying the direction of your sort.
+The direction of your sort can either be **ascending**, or **descending**.
```

Comment:
Nit: remove comma?

Reply:
Good catch. I think you correct in saying this comma should be removed. Fix: https://github.com/mongodb/docs-java/pull/59/commits/b8b78e55e8dc5227b0dfbaeb5af3e9324948bf5d